molecule/default/molecule.yml

```diff
@@ -0,0 +1,209 @@
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper-100:2181
+    published_ports:
+      - "9092"
+    networks:
+      - name: molecule
+        links:
+          - zookeeper-100
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+  # 1.0.1
+  - name: zookeeper-101
+    image: zookeeper:3.4
+    command: "bin/zkServer.sh start-foreground"
+    published_ports:
+      - "2181"
+    networks:
+      - name: molecule
+        aliases:
+          - zookeeper-101
+  - name: kafka1-101
+    image: wurstmeister/kafka:1.0.1
+    command: "start-kafka.sh"
+    env:
+      HOSTNAME_COMMAND: "hostname -i | cut -d' ' -f1"
+      KAFKA_DELETE_TOPIC_ENABLE: "true"
+      KAFKA_ADVERTISED_PORT: "9092"
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper-101:2181
+    published_ports:
+      - "9092"
+    networks:
+      - name: molecule
+        links:
+          - zookeeper-101
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+  - name: kafka2-101
+    image: wurstmeister/kafka:1.0.1
+    command: "start-kafka.sh"
+    env:
+      HOSTNAME_COMMAND: "hostname -i | cut -d' ' -f1"
+      KAFKA_DELETE_TOPIC_ENABLE: "true"
+      KAFKA_ADVERTISED_PORT: "9092"
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper-101:2181
+    published_ports:
+      - "9092"
+    networks:
+      - name: molecule
+        links:
+          - zookeeper-101
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+  # 1.1.0
+  - name: zookeeper-110
+    image: zookeeper:3.4
+    command: "bin/zkServer.sh start-foreground"
+    published_ports:
+      - "2181"
+    networks:
+      - name: molecule
+        aliases:
+          - zookeeper-110
+  - name: kafka1-110
+    image: wurstmeister/kafka:1.1.0
+    command: "start-kafka.sh"
+    env:
+      HOSTNAME_COMMAND: "hostname -i | cut -d' ' -f1"
+      KAFKA_DELETE_TOPIC_ENABLE: "true"
+      KAFKA_ADVERTISED_PORT: "9092"
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper-110:2181
+    published_ports:
+      - "9092"
+    networks:
+      - name: molecule
+        links:
+          - zookeeper-110
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+  - name: kafka2-110
+    image: wurstmeister/kafka:1.1.0
+    command: "start-kafka.sh"
+    env:
+      HOSTNAME_COMMAND: "hostname -i | cut -d' ' -f1"
+      KAFKA_DELETE_TOPIC_ENABLE: "true"
+      KAFKA_ADVERTISED_PORT: "9092"
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper-110:2181
+    published_ports:
+      - "9092"
+    networks:
+      - name: molecule
+        links:
+          - zookeeper-110
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+  inventory:
+    group_vars:
+      all:
+        ansible_kafka_supported_versions:
+          - protocol_version: "0.11.0"
+            instance_suffix: "01101"
+          - protocol_version: "1.0.0"
+            instance_suffix: "100"
+          - protocol_version: "1.0.1"
+            instance_suffix: "101"
+          - protocol_version: "1.0.1"
```

Comment:
I guess it should be 1.1.0 here

Reply:
When i refactor this code Cf https://github.com/StephenSorriaux/ansible-kafka-admin/blob/master/molecule/kafka_1.1.0/playbook.yml#L15 i had seen that your are using 1.0.1 for kafka 1.1.0.

Seeing that, i found that kafka-python does not seem to support 1.1.0 API yet, so it seem that 1.0.1 might be a good choice. (Cf https://github.com/dpkp/kafka-python/issues/1512)

Reply:
You're right, let's keep it that way for now